integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/car/deployment/test/CAppDeploymentOrderTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+*Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+package org.wso2.carbon.esb.car.deployment.test;
+
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.ArrayUtils;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.extensions.servers.ftpserver.FTPServerManager;
+import org.wso2.carbon.integration.common.admin.client.ApplicationAdminClient;
+import org.wso2.carbon.integration.common.admin.client.CarbonAppUploaderClient;
+import org.wso2.carbon.integration.common.admin.client.LogViewerClient;
+import org.wso2.carbon.logging.view.stub.LogViewerLogViewerException;
+import org.wso2.carbon.logging.view.stub.types.carbon.LogEvent;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import org.wso2.esb.integration.common.utils.Utils;
+
+import javax.activation.DataHandler;
+import java.io.File;
+import java.net.URL;
+import java.rmi.RemoteException;
+import java.util.Calendar;
+import java.util.Stack;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class tests whether the artifacts in deployed car file is in order
+ * Related issue: https://github.com/wso2/product-ei/issues/1261
+ */
+public class CAppDeploymentOrderTest extends ESBIntegrationTest {
+    private CarbonAppUploaderClient carbonAppUploaderClient;
+    private ApplicationAdminClient applicationAdminClient;
+    private final int MAX_TIME = 120000;
+    private final String carFileName = "esb-deployment-car_1.0.0";
```

Comment:
the constant variable name should be in block capitals

Reply:
Fixed in fd4486d8cd20c8944413168817517ebb691f3594